src/Makefile

```
@@ -325,10 +325,19 @@ ifndef V
         @printf '    %b %b\n' $(LINKCOLOR)INSTALL$(ENDCOLOR) $(BINCOLOR)$(1)$(ENDCOLOR) 1>&2
         @$(INSTALL) $(1) $(2)
     endef
+
+    define CREATE_SYMLINK
+        @ln -sf $(1) $(2)/$(3)
+        @printf '    %b %b %b %b %b %b %b\n' $(LINKCOLOR)CREATE SYMLINK$(ENDCOLOR) from $(BINCOLOR)$(3)$(ENDCOLOR) to $(BINCOLOR)$(1)$(ENDCOLOR)  1>&2
+    endef
 else
     define MAKE_INSTALL
         $(INSTALL) $(1) $(2)
     endef
+
+    define CREATE_SYMLINK
+        @ln -sf $(1) $(2)/$(3)
```

This is verbose mode, so don't hide the command.

```suggestion
        ln -sf $(1) $(2)/$(3)
```